### PR TITLE
Add new language support in settings configuration

### PR DESCRIPTION
**Pull request by mrvkino**
Hi,

to support new language not available in the code, I have added the possibility to add them in the settings. It will also be useful to support override if this feature is wanted.

In this pull request, I have put 3 format for the settings.

- V1: more verbose, this clarifies the values being used
- V2: more compact, but is the easiest to write and view
- V3: shorten the language and stay verbose on the ambiguous string values

The readme has some example of the format for each. All three configurations format are functional and replace the default limiter for the time being.

Personally, I am leaning toward the V3 format.

I can edit the code to reflect the final format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,43 @@ or in html files
...
+
+```json
+"comment-divider.languagesAssociationsV2": [
+  {
+      "toml": ["#", "#"],
+  }
+]
+```
+
+The item name is the language mode name and is associated with an array of 1 or 2 elements. The first element is the start of the line. The second, if defined, is the end. If the second element is not defined the first element is used to terminate the line.
```
Comment (stackbreak) [inline README.md:149]: There is a need to change this behavior a little. If the **second** element is not defined the line will be without an end limiter. It is suitable for this issue #3 .

**Review by stackbreak — COMMENTED** [review]
@mrvkino great job! 
Please, take a look at the review comments about the format.

File: README.md [hunk 2]
```diff
@@ -122,6 +122,43 @@ or in html files
...
+```
+
+The item language take the language mode name. The language and start_string are required. If the end_string is not defined, the start_string will be used to terminate the line.
+
+```json
+"comment-divider.languagesAssociationsV2": [
+  {
+      "toml": ["#", "#"],
+  }
+]
```
[the remark this answers is not in the document]
Comment (mrvkino) [reply README.md]: Perfect! Version2 it is. It's true, the other option might of gotten too verbose in the long run. The README had a typo, there is no array. I have fixed it in the README when removing v1 and v3. see commit a4f503cce696533406bf0956cac1fd60bbdc3363

File: README.md (hunk 1 above)
Comment (mrvkino) [reply README.md]: I will get on this. Something breaks if I leave the second element as an empty string or undefined. I will fix the code for that.

Also, for issue #3, I might use the result of the switch case as the default value of the function `getLanguageCommentStr` and allow it to override the value if the user define the language in the settings.

Comment (mrvkino) [reply README.md]: It is now possible to specify one element. If the second element is omitted the fille string is used until the end of the line.

Also, a user can override the default limiters used by the extension. The user simply adds the language with the new characters in the `comment-divider.languagesAssociations` configuration.

**Pull request merged**